firebase/initFirebase.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable no-unused-vars */
+import { initializeApp } from 'firebase/app';
+import { getFirestore } from 'firebase/firestore';
+// import { getAnalytics } from 'firebase/analytics';
+
+// TODO: Add SDKs for Firebase products that you want to use
+
+// https://firebase.google.com/docs/web/setup#available-libraries
+
+// Your web app's Firebase configuration
+
+// For Firebase JS SDK v7.20.0 and later, measurementId is optional
+
+const firebaseConfig = {
+  apiKey: 'AIzaSyCDvvTB9u26lnFzwJjFNZJ6OGXUX_XxdFE',
+
+  authDomain: 'wiki-d6b95.firebaseapp.com',
+
+  projectId: 'wiki-d6b95',
+
+  storageBucket: 'wiki-d6b95.appspot.com',
+
+  messagingSenderId: '589888164658',
+
+  appId: '1:589888164658:web:cfa3094ae2d6337823fd2b',
+
+  measurementId: 'G-DLDL7X8BP8',
+};
+
+/* const firebaseConfig = {
+  apiKey: 'AIzaSyAFKUWGP6VjrH5kgdEoZafGMbLzxPqjWLo',
+
+  authDomain: 'hiswiki.firebaseapp.com',
+
+  databaseURL: 'https://hiswiki-default-rtdb.firebaseio.com',
+
+  projectId: 'hiswiki',
+
+  storageBucket: 'hiswiki.appspot.com',
+
+  messagingSenderId: '340390678486',
+
+  appId: '1:340390678486:web:e2997123487291df3e1e77',
+
+  measurementId: 'G-RTGZX9LE9E',
+}; */
+
+const app = initializeApp(firebaseConfig);
```

Comment:
If we can call the `getFirestore` without the `app` - add this logic to _app (above the component). So it is obvious what is going on. No need to assign it anywhere. Just do

```
const firebaseConfig = {...}

initializeApp(firebaseConfig);

const MyApp = () => ....
```

Reply:
If I wrote `initializeApp(firebaseConfig); `in initFirebase.js, and `const db = getFirestore();` in firebase-firsttry.js it threw an error.

Reply:
![Snímek obrazovky 2022-04-10 v 15 42 28](https://user-images.githubusercontent.com/65444975/162621079-49dfa347-8757-4459-8722-948023e1575f.png)

Reply:
That's interesting :D